packages/icons/icons.json

```
@@ -1486,126 +1486,126 @@
+        "tags": "scale, typography, size, a, letter, alphabet, letter a, area",
         "group": "editor",
         "codepoint": 61887
     },
     {
         "displayName": "Header",
         "iconName": "header",
-        "tags": "typography, section, layout",
+        "tags": "typography, section, layout, intersections, letter h, track segment, hotel, helicopter, helipad, capital h, headline",
         "group": "editor",
         "codepoint": 61925
     },
     {
         "displayName": "Saved",
         "iconName": "saved",
-        "tags": "document, check mark, tick",
+        "tags": "document, check mark, tick, approval, document ready, checks complete, checklist, approved document, completed document, verified document, confirm, valid, save, upload finished, confirmation, validation, checked, approved, done document, completed, certificate",
         "group": "file",
         "codepoint": 62109
     },
     {
         "displayName": "Floppy disk",
         "iconName": "floppy-disk",
-        "tags": "save",
+        "tags": "save, disk, save as, disk drive, disc drive",
         "group": "interface",
         "codepoint": 61873
     },
     {
         "displayName": "Book",
         "iconName": "book",
-        "tags": "pages, album, brochure, manual",
+        "tags": "pages, album, brochure, manual, bookmark, notebook, journal, documentation, reading, instructions, file, cathedral",
         "group": "miscellaneous",
         "codepoint": 61750
     },
     {
         "displayName": "Hand: right",
         "iconName": "hand-right",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, hand, point right, finger, tip, attention",
         "group": "interface",
         "codepoint": 61918
     },
     {
         "displayName": "Hand: up",
         "iconName": "hand-up",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, pointer, forefinger, right hand, click, point, cursor",
         "group": "interface",
         "codepoint": 61919
     },
     {
         "displayName": "Hand: down",
         "iconName": "hand-down",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, point down, hand, pointer finger, hand pointing down, select, down",
         "group": "interface",
         "codepoint": 61916
     },
     {
         "displayName": "Hand: left",
         "iconName": "hand-left",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, point, finger, this, here, click",
         "group": "interface",
         "codepoint": 61917
     },
     {
         "displayName": "Thumbs: up",
         "iconName": "thumbs-up",
-        "tags": "hand, like, ok",
+        "tags": "hand, like, ok, good, okay",
         "group": "interface",
         "codepoint": 62189
     },
     {
         "displayName": "Thumbs: down",
         "iconName": "thumbs-down",
-        "tags": "hand, dislike, bad",
+        "tags": "hand, dislike, bad, thumb down",
```

```suggestion
        "tags": "hand, dislike, bad",
```

this addition doesn't change the search results